src/patternfly/components/Popover/popover.scss

```diff
@@ -94,6 +96,36 @@
       transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
     }
   }
+
+  &.pf-m-left.pf-m-start,
+  &.pf-m-right.pf-m-start {
+    .pf-c-popover__arrow {
```

Comment:
If we setup base vars for the arrow's position, we can remove this `.pf-c-popover__arrow` selector and set the vars with the `.pf-m-[direction]` selectors.

Also re: having `.pf-m-left.pf-m-start` vs `.pf-m-top-left`, one argument for the latter is the specificity of the rules will be lower. But seems like that would be more code since you'd need 8 selectors instead of 4.

wdyt @mattnolting?

Reply:
FWIW, I looked at how specific it already was, and what the element is, and my decision was that one more modifier wasn't that significant. This seemed cleaner to implement as well as to understand. But I see how it could go either way.

Reply:
I think `.pf-m-top/bottom-left/right` is consistent with the way CSS defines specific props, like `border-top-left-radius`. That seems a bit more straightforward to me.

Reply:
So @mcoker and @mattnolting, you want `.pf-m-top/bottom/left/right-top/bottom/left/right` and not `.pf-m-top/bottom/left/right-start/end` like popper has, is that correct?